Amazon.IonDotnet.Tests/Internals/ReaderLocalTableTest.cs

```diff
@@ -13,14 +13,22 @@
  * permissions and limitations under the License.
  */
 
+using System;
+using Amazon.IonDotnet.Builders;
 using Amazon.IonDotnet.Internals;
+using Amazon.IonDotnet.Tree;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Amazon.IonDotnet.Tests.Internals
 {
     [TestClass]
     public class ReaderLocalTableTest
     {
+        internal const int UNKNOWN_SYMBOL_ID = -1;
```

Comment:
Can remove and use `SymbolToken.UnknownSid` instead throughout the code.

Reply:
Updated